app/src/main/kotlin/com/mapbox/vision/teaser/utils/FragmentUtils.kt

```diff
@@ -20,8 +20,10 @@ fun Fragment.requireVisionManager(): BaseVisionManager? {
 }
 
 fun Fragment.runOnUiThreadIfPossible(action: () -> Unit) {
```

Comment:
Nit : how about `runOnUiThreadIfResumed` ?
Just found myself a bit confused with the question `when is it possible?`

Reply:
:)
lets use IfResumed, it comes from previous use-cases

Reply:
fixed